src/consumption.cpp

```diff
@@ -161,6 +219,13 @@ edible_rating player::can_eat( const item &food, bool interactive, bool force )
         return INEDIBLE;
     }
 
+    for( const auto &m : food.type->materials ) {
+        if( !m.obj().edible() ) {
```

Comment:
How do we define `COMESTIBLE`? The alternative is an `INEDIBLE` flag but that of course depends upon what the definition of `COMESTIBLE` is?